assets/styles.scss

```diff
@@ -71,3 +71,79 @@ html.has-custom-scrollbars {
     border-color: transparent;
   }
 }
+
+html.has-light-theme {
+  background-color: #F8F8F8;
+
+  body {
+    color: #333;
+  }
+
+  h1.title {
+    text-shadow: 1px 0px #ffdd57;
+  }
+
+  .has-underline {
```

Comment:
Are you familiar with Vue? Can you move this to Vue component page instead? Something like 

```
.has-underline {
  border-bottom: 1px solid #fff;
  padding: 1em 0px;
  margin-bottom: 1em;

  html.has-light-theme & {
    border-bottom: 1px solid #DCDCDC;
  }
}

```